client/sw.js

```diff
@@ -0,0 +1,48 @@
+// In production, we register a service worker to serve assets from local cache.
```

Comment:
For multiple lines of comment, it is much preferable to use this `/** */`.

Reply:
Thank you. 👍